src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -312,12 +363,37 @@ val qualityFormat: Option[FastqQualityFormat] = None,
     // Close the writer; NB: the inputs close automatically
     sampleInfos.foreach(_.writer.close())
 
+    // Optionally create the FASTQs
+    if (outputFastqs) {
```

Comment:
Ok, this feels _really_ weird to me.  I figured the idea here was to add an option to output _either_ BAMs or FASTQs.  I find it hard to believe that anyone would really want both - i.e. if I prefer to demux to fastq files, then I'm probably going to ignore the BAM files and vice versa.  Shouldn't the options be exclusive?